datafusion/core/src/dataframe/mod.rs

```diff
@@ -534,7 +534,7 @@ impl DataFrame {
                 vec![],
                 original_schema_fields
                     .clone()
-                    .map(|f| count(is_null(col(f.name()))).alias(f.name()))
+                    .map(|f| count_null(col(f.name())).alias(f.name()))
```

Comment:
Instead of a `count_null()` new function, what do you think about using the existing [`count`](https://docs.rs/datafusion/latest/datafusion/logical_expr/fn.count.html) function?

Something like `count(expr.is_null())`

```suggestion
                    .map(|f| count(col(f.name()).is_null()).alias(f.name())))
```

Reply:
When using `count`, it sums all `non-null` Expr while `is_null` converts `Expr` to a boolean expression, which is why it doesn't work.